ds3_java_cli/src/main/java/com/spectralogic/ds3cli/views/cli/GetSystemFailureView.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * ******************************************************************************
+ *   Copyright 2014 - 2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3cli.views.cli;
+
+import com.bethecoder.ascii_table.ASCIITable;
+import com.google.common.collect.ImmutableList;
+import com.spectralogic.ds3cli.models.GetSystemFailureResult;
+import com.spectralogic.ds3client.models.SystemFailure;
+import com.spectralogic.ds3client.models.SystemFailureList;
+
+import java.util.List;
+
+import static com.spectralogic.ds3cli.util.Utils.*;
+
+public class GetSystemFailureView extends TableView<GetSystemFailureResult> {
+
+    protected List<SystemFailure> failures;
+
+    @Override
+    public String render(final GetSystemFailureResult obj) {
+        final SystemFailureList result = obj.getResult();
+        if( (result == null)
+                || (null == result.getSystemFailures())
```

Comment:
Do I need to first test for null result, or can I be sure that the client would have caught that?